### ISSUE TITLE
Re-load of dataset is taking a long time

### ISSUE THREAD

**Issue by jellypuno** [zowe/vscode-extension-for-zowe#471]
This is related to issue #327 

I have a dataset with 30k members and when I try to do a re-load, it loads at least 5 minutes and then ends in error:

![Capture3](https://user-images.githubusercontent.com/45042840/73162452-1cee6100-40ee-11ea-98a7-88a1a0b21bd2.PNG)

**Comment by MikeBauerCA** [comment]
@jellypuno is this issue still occurring? I noticed during one of your recent presentations, you were able to load a large PDS fairly quickly. This was just initial search though, not reload.

**Comment by jellypuno** [comment]
@MikeBauerCA Yup. It is still happening if I do Step#2. Loading/Refreshing of the tree is still taking a long time.

**Comment by JillieBeanSim** [comment]
fixed in PR https://github.com/zowe/vscode-extension-for-zowe/pull/1914
